scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -46,6 +46,21 @@ class H2OMOJOModel(override val uid: String) extends H2OMOJOModelBase[H2OMOJOMod
 
   def getModelDetails(): String = $(modelDetails)
 
+  def getDomainValues(): Array[Array[String]] = {
```

Comment:
Shouldn't we return `Map[String, Array[String]]` where key is a column name instead?

Reply:
Yup, better idea!

Reply:
With this I think we should keep only one method `def getDomainValues(): Map[String, Array[String]]`, users can extract the specific column themselves

Reply:
Yup, makes sense!